grpcbp/server.go

```diff
@@ -0,0 +1,128 @@
+package grpcbp
+
+import (
+	"net"
+	"time"
+
+	"github.com/reddit/baseplate.go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+)
+
+// ServerConfig is the argument struct for NewBaseplateServer. Please refer to
+// the documentation for each field to see how is it used.
+type ServerConfig struct {
+	// MaxConnectionIdle is a duration for the amount of time after which an idle
+	// connection would be closed by sending a GoAway. Idleness duration is
+	// defined since the most recent time the number of outstanding RPCs became
+	// zero or the connection establishment. The current default value is
+	// infinity.
+	MaxConnectionIdle time.Duration `yaml:"maxConnectionIdle"`
+
+	// MaxConnectionAge is a duration for the maximum amount of time a
+	// connection may exist before it will be closed by sending a GoAway. A
+	// random jitter of +/-10% will be added to MaxConnectionAge to spread out
```

Comment:
I'm not sure adding the jitter on the server side is useful.. unless the client will auto reconnect once it lose the connection.

Reply:
From my understanding, it will.